docs/core/diagnostics/available-counters.md

```diff
@@ -11,7 +11,7 @@ The .NET runtime and libraries implement and publish several [`EventCounter`](./
 
 ## System.Runtime counters
 
-The following counters are published as part of .NET runtime (CoreCLR) and are maintained in the [`RuntimeEventSource.cs`](https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSource.cs).
+The following counters are published as part of .NET runtime (CoreCLR) and are maintained in the [`RuntimeEventSource.cs`](https://github.com/dotnet/coreclr/blob/main/src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSource.cs).
```

Comment:
Should this link to runtime repository instead?

```suggestion
The following counters are published as part of .NET runtime (CoreCLR) and are maintained in the [`RuntimeEventSource.cs`](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/RuntimeEventSource.cs).
```

@stephentoub Can you confirm?

Reply:
Yes, the coreclr repo is only for servicing older versions.